libraries/plugins/peerplays_sidechain/addresses.txt

```diff
@@ -0,0 +1,391 @@
+    "pubkey": "022752cd513f04f68074bd90a96a82b523a197171376382dedf3413bbdccae0dac",
+    "address": "bcrt1qxkm4klp55lus392m3wf64fnh4frl4vkyng4ffg",
+    "scriptPubKey": "001435b75b7c34a7f908955b8b93aaa677aa47fab2c4"
+  },
+  "label": "",
+  "ischange": false,
+  "timestamp": 1571845292,
+  "hdkeypath": "m/0'/0'/9'",
+  "hdseedid": "7fc94a298c785719434c63458bb41bc1995a270d",
+  "hdmasterfingerprint": "153472fd",
+  "labels": [
+    {
+      "name": "",
+      "purpose": "receive"
+    }
+  ]
+}
+
+2N1sFbwcn4QVbvjp7yRsN4mg4mBFbvC8gKM
+{
+  "address": "2N1sFbwcn4QVbvjp7yRsN4mg4mBFbvC8gKM",
+  "scriptPubKey": "a9145e91543069ae37d8bace2f59aade945f5916dc3287",
+  "ismine": true,
+  "solvable": true,
+  "desc": "sh(wpkh([153472fd/0'/0'/10']03114bf9794439221c0f7e910d7b64f242847184381a5ef238cef8d70f8868b4af))#x7xpvc0y",
+  "iswatchonly": false,
+  "isscript": true,
+  "iswitness": false,
+  "script": "witness_v0_keyhash",
+  "hex": "0014ceb35b8198b58c76fcc78a70d1d3a2ed1d4325f2",
+  "pubkey": "03114bf9794439221c0f7e910d7b64f242847184381a5ef238cef8d70f8868b4af",
+  "embedded": {
+    "isscript": false,
+    "iswitness": true,
+    "witness_version": 0,
+    "witness_program": "ceb35b8198b58c76fcc78a70d1d3a2ed1d4325f2",
+    "pubkey": "03114bf9794439221c0f7e910d7b64f242847184381a5ef238cef8d70f8868b4af",
+    "address": "bcrt1qe6e4hqvckkx8dlx83fcdr5aza5w5xf0jd22xet",
+    "scriptPubKey": "0014ceb35b8198b58c76fcc78a70d1d3a2ed1d4325f2"
+  },
+  "label": "",
+  "ischange": false,
+  "timestamp": 1571845292,
+  "hdkeypath": "m/0'/0'/10'",
+  "hdseedid": "7fc94a298c785719434c63458bb41bc1995a270d",
+  "hdmasterfingerprint": "153472fd",
+  "labels": [
+    {
+      "name": "",
+      "purpose": "receive"
+    }
+  ]
+}
+
+2NDmxr6ufBE7Zgdgq9hShF2grx2YPEiTyNy
+{
+  "address": "2NDmxr6ufBE7Zgdgq9hShF2grx2YPEiTyNy",
+  "scriptPubKey": "a914e132c578bd294a01a472d42b376b29e5ef6678c987",
+  "ismine": true,
+  "solvable": true,
+  "desc": "sh(wpkh([153472fd/0'/0'/11']02eb6e43fb6ad9f1bf23bf821a25d5a1e84550380e0973ea5b00b087b3e9059c7b))#2janhauz",
+  "iswatchonly": false,
+  "isscript": true,
+  "iswitness": false,
+  "script": "witness_v0_keyhash",
+  "hex": "0014831d63f8d99be71f9b385f2ccc92ab2783da3762",
+  "pubkey": "02eb6e43fb6ad9f1bf23bf821a25d5a1e84550380e0973ea5b00b087b3e9059c7b",
+  "embedded": {
+    "isscript": false,
+    "iswitness": true,
+    "witness_version": 0,
+    "witness_program": "831d63f8d99be71f9b385f2ccc92ab2783da3762",
+    "pubkey": "02eb6e43fb6ad9f1bf23bf821a25d5a1e84550380e0973ea5b00b087b3e9059c7b",
+    "address": "bcrt1qsvwk87xen0n3lxectukvey4ty7pa5dmz0yanuv",
+    "scriptPubKey": "0014831d63f8d99be71f9b385f2ccc92ab2783da3762"
+  },
+  "label": "",
+  "ischange": false,
+  "timestamp": 1571845292,
+  "hdkeypath": "m/0'/0'/11'",
+  "hdseedid": "7fc94a298c785719434c63458bb41bc1995a270d",
+  "hdmasterfingerprint": "153472fd",
+  "labels": [
+    {
+      "name": "",
+      "purpose": "receive"
+    }
+  ]
+}
+
```

Comment:
If file is needed, remove newlines after here.

Reply:
Not needed.